system-tests/test/component_testing_index_html_spec.ts

```diff
@@ -0,0 +1,29 @@
+import systemTests from '../lib/system-tests'
+
+describe('component testing projects', function () {
+  systemTests.setup()
+
+  systemTests.it('create-react-app-custom-index-html', {
+    project: 'create-react-app-custom-index-html',
+    testingType: 'component',
+    spec: 'src/App.cy.jsx',
```

Comment:
More a question for @flotwig, is there some way to say "run all the specs in the project" for a system test project, instead of specifying one here? I tried a few things but didn't get it to work.

Reply:
Omitting the `spec` parameter should do it. Or `*`, `**/*`... it's passed straight to `--spec`, so anything like that should work.